main.js

```diff
@@ -32,36 +33,71 @@ class Field {
       this.playerYPosition += 1;
     }
 
-    if (!this.gameLost()) {
-      // TODO: Insert player indicator in new spot here
+    console.log("X: " + this.playerXPosition);
+    console.log("Y: " + this.playerYPosition);
+
+    if (!this.checkIfGameIsLost(true)) {
+      if (event === "a" || event === "d" || event === "w" || event === "s") {
+        this.field[this.playerYPosition].splice(this.playerXPosition, 1, "*");
```

Comment:
Does the same thing, but simpler:

```suggestion
        this.field[this.playerYPosition][this.playerXPosition] = "*";
```